ci/check-release-notes.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python3
+
+import argparse
+import yaml
+import re
+import sys
+
+def validate_release_notes_structure(data, file_name):
```

Comment:
Instead of taking in `file_name`, we can leave it out of error messages and instead in the for-loop in `main()`, print the filename being validated at the start of each iteration.

Reply:
I have updated the code now for the filename to be printed at the start of each iteration